code/modules/antagonists/demon/objectives.dm

```diff
@@ -0,0 +1,17 @@
+/datum/objective/demonsingeneral
```

Comment:
```suggestion
/datum/objective/demonsingeneral
```
why not just /datum/objective/demon ?

Reply:
i was planning on adding more specific objectives too but ended up not doing it and never changed the name